### ISSUE TITLE
ksql.advertised.listener required?

### ISSUE THREAD

**Issue by CodeGlitcher** [confluentinc/ksql#9155]
Hi 
we are running ksqldb in Kubernetse with a  KSQL_LISTENERS: https://0.0.0.0:8088.
We want to switch from headless mode to interactive mode so we can execute some queries directy from our application.

According to the docs the ksql.advertised.listener is used for inter-node communication.
But we are having trouble settings tgus paramter from kubernetes.

When we create a statefull set we can set de pods dns adress in this field.
But then its not formatted as a url with https://nodename:8088.

And I cant find if a can set the value as:
ksql.advertised.listener=https://<hostname from env variable>:8088.

So in short. Do i need to configure ksql.advertised.listener or can i ignore it?
we want to deploy multiple ksqldb nodes but we want kubernetes to do the scaling.
and we dont want to deploy manually multple ksqldb deployment juist because we need to set this 1 value.

**Comment by CodeGlitcher** [comment]
we found a way to set the ksql.advertised.listener so this problem is solved
